charts/open-notificaties/values.yaml

```diff
@@ -21,16 +21,16 @@ serviceAccount:
 
 podAnnotations: {}
 
-podSecurityContext: {}
-  # fsGroup: 2000
+podSecurityContext:
+  runAsUser: 1000
```

Comment:
I think this is not "required". When `USER` is specified in the Dockerfile, you can leave this empty.

Reply:
yeah wasn't sure about this one, but tools like Gatekeeper *apparently* aren't able to inspect the container `USER` and rely on explicit security contexts specified at the k8s level.

This change would make this "correct by default", but I don't have strong opinions on it.

Reply:
Hmm ok, no problems with leaving this in.